src/Maestro/cluster-template/template.json

```diff
@@ -260,7 +260,7 @@
                     "nicPrefixOverride": "10.0.0.0/24",
                     "certificate": {
                       "commonNames": [
-                        "[reference('network').outputs.fqdn.value]"
+                        "maestro-prod.westus2.cloudapp.azure.com"
```

Comment:
Does does this need to be different for maestro-int?

Reply:
...and local? Not sure exactly what this does

Reply:
Nope. It used to be different, but we normalized to a single SSL cert with alternative names for both int and prod some time ago.  This template wasn't updated when that changeover happened.  We had installed _both_ certs on the scale set, so it wasn't a problem until we paved it (because changing an existing machine didn't delete the existing certs, so everything worked).

The local cluster doesn't use certs for managing things, so this change doesn't apply there.